src/Components/ErrorPage/ErrorPage.css

```diff
@@ -0,0 +1,27 @@
+.error-container {
+    height: 100vh;
+    background-image: url('/src/Assets/error-48252.png');
+    background-position: center;
+    background-size: cover;
+    color: #ff0000;
+    width: 50vw;
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+}
+
+.error-string {
+font-weight: 700;
+}
+
+.homee-btn {
```

Comment:
Is this working? `home` is misspelled so I just wanted to check with ya in case you were like, wtf is going on here?! lol